extensions/mssql/src/localizedConstants.ts

```diff
@@ -55,3 +55,10 @@ export function sparkJobSubmissionGetApplicationIdFailed(err: string): string {
 export function sparkJobSubmissionLocalFileNotExisted(path: string): string { return localize('sparkJobSubmission.LocalFileNotExisted', "Local file {0} does not existed. ", path); }
 export const sparkJobSubmissionNoSqlBigDataClusterFound = localize('sparkJobSubmission.NoSqlBigDataClusterFound', "No SQL Server Big Data Cluster found.");
 export function sparkConnectionRequired(name: string): string { return localize('sparkConnectionRequired', "Please connect to the Spark cluster before View {0} History.", name); }
+
+
+export const failedToFindTenants = localize('mssql.failedToFindTenants', "Failed to find tenants");
+export function tokenRefreshFailed(name: string): string { return localize('mssql.tokenRefreshFailed', "{0} AAD token refresh failed, please reconnect to enable {0}", name); }
+export const tokenRefreshFailedToken = localize('mssql.tokenRefreshFailedToken', "Editor token refresh failed - no security token, please reconnect to enable intellisense");
```

Comment:
`no security token` doesn't really make help or make sense - there's a lot of reasons why the refresh could have failed. 

Also `please reconnect` isn't really correct either. The editor itself isn't disconnected when this happens - users can still run queries. It's just autocompletion that's disabled.

Would suggest 

`Editor token refresh failed, autocompletion will be disabled until the editor is disconnected and reconnected`

But also as mentioned above, this should go into STS and be passed as a parameter to the request